source/features/one-click-review-submission.tsx

```diff
@@ -30,9 +29,13 @@ function replaceCheckboxes(originalSubmitButton: HTMLButtonElement): void {
 		);
 	}
 
+	const formAttribute = originalSubmitButton.getAttribute('form')!;
+
 	// Generate the new buttons
 	for (const radio of radios) {
-		const tooltip = radio.parentElement!.getAttribute('aria-label');
+		const labelNode = radio.labels?.[0];
+
+		const tooltip = labelNode?.nextElementSibling?.textContent;
```

Comment:
So, weirdly it looks from the previous DOM structure like there wasn't an `aria-label` on its parent anyway. I suspect that could have broken a while ago. Not sure exactly what the intended behaviour was before, but I've actually re-removed the hybrid approach because I don't believe it was doing anything

Reply:
This label is grabbing too much on GHE:
```
<label class="d-block">
        <input
          type="radio"
          name="pull_request_review[event]"
          value="comment"
          checked=""
        />
        Comment
        <p class="text-normal color-fg-muted">
          Submit general feedback without explicit approval.
        </p>
      </label>
```
Can do something "fun" like:
```
const label = labelNode?.nextElementSibling;
const nestedDescription = label.getElementsByTaName('p'); // GHE has a nested P with description inside the label
if (nestedDescription.length) {
   label.removeChild(nestedDescription[0]);
}
const tooltip = label.textContent;
```

I'm sure there's a better way to achieve that, but that should get us there.